### ISSUE TITLE
Plugin created using grails 3.0.8 doesn't have grailsVersion instantiated

### ISSUE THREAD

**Issue by bond-** [grails/grails-core#9315]
Reproduction steps:
--
1. grails create-plugin sample
2. cd sample
3. grails compile

and the following error message follows:
```
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'security'.
> Could not resolve all dependencies for configuration ':classpath'.
   > Could not find org.grails:grails-gradle-plugin:.
     Searched in the following locations:
         file:/Users/raviteja.lokineni/.m2/repository/org/grails/grails-gradle-plugin//grails-gradle-plugin-.pom
         file:/Users/raviteja.lokineni/.m2/repository/org/grails/grails-gradle-plugin//grails-gradle-plugin-.jar
         https://repo.grails.org/grails/core/org/grails/grails-gradle-plugin//grails-gradle-plugin-.pom
         https://repo.grails.org/grails/core/org/grails/grails-gradle-plugin//grails-gradle-plugin-.jar
         https://plugins.gradle.org/m2/org/grails/grails-gradle-plugin//grails-gradle-plugin-.pom
         https://plugins.gradle.org/m2/org/grails/grails-gradle-plugin//grails-gradle-plugin-.jar
     Required by:
         :security:unspecified

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 8.937 secs
| Error Error initializing classpath: Could not find org.grails:grails-gradle-plugin:.
Searched in the following locations:
    file:/Users/raviteja.lokineni/.m2/repository/org/grails/grails-gradle-plugin//grails-gradle-plugin-.pom
    file:/Users/raviteja.lokineni/.m2/repository/org/grails/grails-gradle-plugin//grails-gradle-plugin-.jar
    https://repo.grails.org/grails/core/org/grails/grails-gradle-plugin//grails-gradle-plugin-.pom
    https://repo.grails.org/grails/core/org/grails/grails-gradle-plugin//grails-gradle-plugin-.jar
    https://plugins.gradle.org/m2/org/grails/grails-gradle-plugin//grails-gradle-plugin-.pom
    https://plugins.gradle.org/m2/org/grails/grails-gradle-plugin//grails-gradle-plugin-.jar
Required by:
    :security:unspecified (Use --stacktrace to see the full trace)
```

**Comment by graemerocher** [comment]
works fine for me, must be something with your environment.

**Comment by bond-** [comment]
Hmm.. Will check it out.

On Wed, Oct 7, 2015 at 8:06 PM, graemerocher <notifications@github.com>
wrote:

> Closed #9315 <https://github.com/grails/grails-core/issues/9315>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grails/grails-core/issues/9315#event-429193942>.
>



-- 
*Raviteja Lokineni* | Application Developer
ServiceNow | Transform IT

E: raviteja.lokineni@gmail.com

[image: View Raviteja Lokineni's profile on LinkedIn]
<http://in.linkedin.com/in/ravitejalokineni>

**Comment by bond-** [comment]
Added ref to issue: 9487 by mistake. This is the same as #9488.
